README.md

```diff
@@ -12,8 +12,12 @@ $ bin/setup
 
 Run the API locally:
 
+It requires a AWS credential that have access to the S3 object storing the public settings of the pan-domain-authentication.  Many accounts such as workflow and targeting have the access.  You may need to set AWS_PROFILE environment to choose which profile to use before running it locally.
+
+For example, if you want to use the profile "workflow" for the AWS credential, you may run:
```

Comment:
Good point to mention Workflow and Targeting account access should work. I would tend to assume the credentials for this project would be from the "Targeting"  AWS account.